blog/models.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    class Meta:
+        verbose_name = "user"
+        verbose_name_plural = "users"
+
+    def __str__(self):
+        return f"{self.username}"
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(User,
+                              on_delete=models.CASCADE,
+                              related_name="posts")
+    title = models.CharField(max_length=200)
```

Comment:
You might want to add a `blank=False, null=False` to the `title` field. This is to ensure that when creating a Post object, the title must be provided and it can't be empty.